sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -873,9 +874,20 @@ class SessionCatalog(
       throw new IllegalStateException("Invalid view without text.")
     }
     val viewConfigs = metadata.viewSQLConfigs
+    val origin = Origin(
+      line = Some(0),
+      startPosition = Some(0),
+      startIndex = Some(0),
+      stopIndex = Some(viewText.length - 1),
+      sqlText = Some(viewText),
+      objectType = Some("VIEW"),
+      objectName = Some(metadata.qualifiedName)
```

Comment:
I think we only need to set `objectType` and `objectName` here? The parser should take care of the other fields.